pkg/api/clients/statestore/statestore.gen.go

```diff
@@ -0,0 +1,1108 @@
+// Package statestore provides primitives to interact with the openapi HTTP API.
```

Comment:
I think the generated open API code belongs in the original schema repo imo. This ensures the code is up to date in CI and has no drift.

It also means others can import the clients, without needing to import `dapr/dapr`. Might also make sense to move `pkg/apis...` there too.

Reply:
I agree that the client code does not belong to the server code.